### DISCUSSION TITLE
Importing GIFs

### DISCUSSION THREAD

**Discussion by ghost** [Orama-Interactive/Pixelorama#584] [Ideas]
unable to import a gif to edit it. can export as gif but not import or edit. 

editing gifs would make this very wonderful

**Comment by OverloadedOrama** [comment]
Gif and video importing has been implemented in c9ddde719493588583a0742138a104e3131ef03e and it will be a part of update 1.0. Keep in mind that this solution will require FFMPEG to be installed in your device.
